File: easy/plusone.py
def plusOne(digits):
        """
        :type digits: List[int]
        :rtype: List[int]
        """
        digs = [str(i) for i in digits]
        num = ''.join(digs)
        num = int(num) + 1
        return [int(i) for i in str(num)]



digits = [1,2,3]
print(plusOne(digits))

File: easy/plusOne.java
public int[] plusOne(int[] digits) {
  for (int i = digits.length - 1; i >= 0; i--) {
      if (digits[i] == 9) {
          if (i == 0) {
              int[] ans = new int[digits.length+1];
              ans[i] = 1;
              ans[i+1] = 0;
              for (int j = 2; j < ans.length; j++) {
                  ans[j] = digits[j-1];
              }
              return ans;
          }
          digits[i] = 0;
      } else {
          digits[i] = digits[i] + 1;
          return digits;
      }
  }
  return digits;
}